src/qt/bitcoingui.cpp

```diff
@@ -680,6 +688,18 @@ void BitcoinGUI::addWallet(WalletModel* walletModel)
     }
     const QString display_name = walletModel->getDisplayName();
     m_wallet_selector->addItem(display_name, QVariant::fromValue(walletModel));
+
+    connect(wallet_view, &WalletView::outOfSyncWarningClicked, walletFrame, &WalletFrame::outOfSyncWarningClicked);
```

Comment:
nit, maybe this should stay in `WalletFrame`?

Reply:
Doing in this way keeps compatibility with #333.